src/plot.jl

```diff
@@ -504,3 +504,21 @@ function Base.show(io::IO, p::Plot)
         print_title(io, border_padding, p.xlabel, p_width = border_length)
     end
 end
+
+"""
+    savefig(p, fn[, color])
+
+Save a `Plot` to a file. The optional keyword argument `color` determines if the ANSI color codes
+should be included or not in the file.
+
+# Examples
+```julia-repl
+julia> savefig(lineplot([0, 1]), "foo.txt")
+
+```
+"""
+function savefig(p::Plot, fn::String; color::Bool=false)
```

Comment:
I would imagine users writing `savefig(p, "out.png")`  and expect some png format results. Since we don't support it we should probably add a warning on some known extensions so that users don't get surprised when they try to open the file with their favorite image reader.

```suggestion
function savefig(p::Plot, fn::String; color::Bool=false)
    ext = lowercase(splitext(fn)[2])
    ext in (".png", ".jpg", ".jpeg", ".tif", ".gif", ".svg") && @warn "`UnicodePlots.savefig` only support writing to text files" filename=fn
```